embulk-standards/src/main/java/org/embulk/standards/LocalFileInputPlugin.java

```diff
@@ -43,6 +47,10 @@
         @ConfigDefault("null")
         Optional<String> getLastPath();
 
+        @Config("follow_symlinks")
+        @ConfigDefault("false")
+        Boolean getFollowSymlinks();
```

Comment:
"boolean" is better than "Boolean" as the return type in general. Do you have any reasons why you use it?